tests/integration/article.test.js

```diff
@@ -38,6 +50,8 @@ describe("# Articles endpoints", () => {
         .set("Authorization", `Bearer ${token}`)
         .send(newArticle)
         .end((err, res) => {
+          createdArticle = res.body;
+          validArticleId = res.body.article.id;
```

Comment:
@niomwungeri-fabrice , you can use destructuring here

Reply:
If you want a different variable name compared to what the object has, it not necessary to destructure an object. right? @espoirMur